src/Util.h

```diff
@@ -57,23 +57,23 @@ std::string GetAuthToken(uWS::HttpRequest* http_request);
 
 // ************ structs *************
 //
-// Usage of the ChannelRange:
+// Usage of the AxisRange:
```

Comment:
It's a little confusing that this is called `AxisRange` and not `ZAxisRange` or `ZRange`, if it is used only for the Z axis (the problem is the hardcoded end in the argument-less constructor).

Reply:
OK, I see now that the end isn't actually hardcoded to the depth; it's hardcoded to the constant for using the full axis which is later translated to the depth. So I guess my issue is the slightly inconsistent naming of `ALL_Z` and `AxisRange`, both of which are actually completely axis-agnostic, but are only used for the Z axis. But this is not a huge problem.

Reply:
I made the struct name generic since it could be used for any axis including stokes, e.g. refactor Frame::GetImageSlicer with a stokes range. ALL_Z was an incomplete refactor of this struct; the axis constants should be generic instead of redundant DEFAULT, CURRENT, and ALL constants for Z and STOKES.  I will do that in a new branch since this one has been merged.